dev/test-studio/sanity.config.ts

```diff
@@ -79,19 +71,43 @@ export default createConfig([
   {
     name: 'default',
     title: 'Test Studio',
-    logo: Branding,
     projectId: 'ppsg7ml5',
     dataset: 'test',
     plugins: [sharedSettings()],
     basePath: '/test',
+    studio: {
+      components: {
+        Logo: Branding,
```

Comment:
I know this is just the `test-studio`, but would be good to rename `Branding` to `Logo` imo.